docs/sdba.rst

```diff
@@ -117,3 +114,15 @@ Developer tools
 
 .. autofunction:: xclim.sdba.measures.check_same_units_and_convert
    :noindex:
+
+.. _sdba-footnotes:
+
+SDBA Footnotes
+==============
+
+.. bibliography::
+   :style: unsrt
+   :labelprefix: SDBA
+   :keyprefix: sdba-
+
+.. [RRJF2021] Roy, P., Rondeau-Genesse, G., Jalbert, J., Fournier, É. 2021. Climate Scenarios of Extreme Precipitation Using a Combination of Parametric and Non-Parametric Bias Correction Methods. Submitted to Climate Services, April 2021.
```

Comment:
Yes. Once we have a DOI for the paper, we should add it to the references. For now, it's defined in the SDBA references so that it doesn't get rendered in multiple pages.